pretty_cron/api.py

```diff
@@ -106,7 +121,17 @@ def _human_list(listy):
 
 
 def _human_week_day(day):
-    return _WEEKDAYS[day]
+    """
+    Cron-tab to weekday
```

Comment:
As before, you are right. Do you prefer me to remove all the docstrings or write them in the right way explaining parameters and return stuff?